### PR TITLE
Fix drawing with dynamic rendering in secondary command buffers

### PR DESCRIPTION
**Pull request by ziga-lunarg**

### CODE REVIEW

File: tests/vklayertests_command.cpp [hunk 1]
```diff
@@ -11338,3 +11338,73 @@ TEST_F(VkLayerTest, ResolveInvalidUsage) {
...
+    pipeline_rendering_info.colorAttachmentCount = 1;
+    pipeline_rendering_info.pColorAttachmentFormats = &format;
+
+    CreatePipelineHelper pipe(*this);
+    pipe.InitInfo();
+    pipe.gp_ci_.pNext = &pipeline_rendering_info;
+    pipe.InitState();
+    pipe.CreateGraphicsPipeline();
+
+    const VkCommandBufferInheritanceRenderingInfoKHR inheritanceRenderingInfo = {
```
Comment (ncesario-lunarg) [inline tests/vklayertests_command.cpp:11381]: I think we're trying to use the `LvlInitStruct` helper for initializing these structs in general (that also makes it a little easier to read which fields are initialized to what IMHO).

**Review by ncesario-lunarg — APPROVED** [review]
LGTM!

**Review by jeremyg-lunarg — APPROVED** [review]
LGTM

File: tests/vklayertests_command.cpp (hunk 1 above)
Comment (ziga-lunarg) [reply tests/vklayertests_command.cpp:11381]: Missed this because it was copy pasted from CTS

**Pull request merged**